node/channel.js

```diff
@@ -453,20 +454,37 @@ TChannel.prototype.makeSubChannel = function makeSubChannel(options) {
     return chan;
 };
 
-TChannel.prototype.listen = function listen(port, host, callback) {
+TChannel.prototype.listen = function listen(port, host, opts, callback) {
```

Comment:
s/opts/options/ (we have been moving toward using options consistently, please bear with the inconsistencies that remain)